tree/ntuple/v7/src/RColumn.cxx

```diff
@@ -46,7 +46,9 @@ void ROOT::Experimental::Detail::RColumn::Connect(DescriptorId_t fieldId, RPageS
    case EPageStorageType::kSink:
       fPageSink = static_cast<RPageSink*>(pageStorage); // the page sink initializes fHeadPage on AddColumn
       fHandleSink = fPageSink->AddColumn(fieldId, *this);
-      fHeadPage = fPageSink->ReservePage(fHandleSink);
+      fApproxNElementsPerPage = fPageSink->GetWriteOptions().GetApproxUnzippedPageSize() / fElement->GetSize();
+      fHeadPage[0] = fPageSink->ReservePage(fHandleSink, fApproxNElementsPerPage + fApproxNElementsPerPage / 2);
+      fHeadPage[1] = fPageSink->ReservePage(fHandleSink, fApproxNElementsPerPage + fApproxNElementsPerPage / 2);
```

Comment:
Please make sure that this is well-behaved for `fApproxNElementsPerPage`. Consider adding a test for this?

Reply:
It's now enforcing `fApproxNElementsPerPage >= 2`